src/components/stateless/specific/pages/ProductPage/ProductPage.jsx

```diff
@@ -9,6 +10,17 @@ import { loadImage } from '../../../../../utils/loadFromIPFS';
 import BuyProduct from './BuyProduct/BuyProduct';
 import SummaryApprovedProduct from './SummaryApprovedProduct/SummaryApprovedProduct';
 import SummaryProduct from './SummaryProduct/SummaryProduct';
+import { fetchAttributes } from '../../../../../redux/actions/AttributesAction.js';
+
+const mapStateToProps = state => ({
+  attributes: state.attributes,
```

Comment:
Name might be confusing, are we talking `requirementsAttributes` or `measurementsAtrtributes`? Let's put name it either `requirementsAttributes` or simply `requirements`